api/server.go

```diff
@@ -108,6 +115,7 @@ func StartAPI(registry *plugin.Registry, mountpoint string, socketPath string) (
 
 	r := mux.NewRouter()
 
+	r.Handle("/analytics/screenview", screenviewHandler).Methods(http.MethodPost)
```

Comment:
I find it kind of annoying that this shows up in the activity journal:
```
Aug  6 14:37:56.528 API: POST /analytics/screenview
Aug  6 14:37:56.532 API: POST /analytics/screenview complete
```

Reply:
I don't think it's that bad. But we could ignore "/analytics" endpoint pings if it is annoying.